src/components/molecules/ListItem.js/ListItem.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Button from 'components/atoms/Button/Button';
+import { Wrapper, Image, Number, Description, AlbumTitle, ArtistName } from './ListItem.styles';
+
+export const ListItem = ({ el, index }) => {
+  return (
+    <Wrapper>
```

Comment:
Jako, że jest to element listy, zawsze warto pamiętać o dodaniu "key": https://pl.reactjs.org/docs/lists-and-keys.html#keys

Największe znaczenie ma to w sytuacjach, gdy lista jest edytowalna, natomiast dobrą praktyką jest zawsze o tym pamiętać. Hint: z API każdy element ma pole "id" - takie id jest unikatowe i najczęsciej sprawdza się bardzo dobrze własnie jako klucz elementu listy.

To jedyna rzecz do której bym realnie mogła się przyczepić i warto poprawić :)